modules/spout/renderablespherespout.cpp

```diff
@@ -31,40 +31,28 @@
 #include <openspace/util/sphere.h>
 #include <ghoul/opengl/texture.h>
 
+namespace {
```

Comment:
You can also add a description on top of the Parameters to describe the entire renderable. It will appear as the description of the renderable in the documentation:) For example, a slight rewrite of the asset description